models/public/cocosnet/cocosnet.md

```diff
@@ -0,0 +1,129 @@
+# cocosnet (composite)
+
+## Use Case and High-Level Description
+
+Cross-domain correspondence network is a exemplar-based image translation composite model, consisting of correspondence and translation parts. Model was pre-trained on ADE20k dataset.
+For details see [paper](https://arxiv.org/pdf/2004.05571) and [repository](https://github.com/microsoft/CoCosNet).
+
+## Example
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove empty section